### PR TITLE
use uwsgi to serve API

### PR DESCRIPTION
**Pull request by flepied**

### CODE REVIEW

File: monocle/webapp.py [hunk 1]
```diff
@@ -150,14 +161,6 @@ def indices():
 
 
 def main():
-    config_path = os.getenv('CONFIG', None)
```
Comment (morucci) [inline monocle/webapp.py:153]: Why not keeping this as a function. I'd like to have the authorization support when I run the flask dev server outside of docker or w/o the WSGI server

File: doc/adr/0006-cache-es-queries.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+distributed solution if needed in the future.
+
+## Considered Options
+
+* [option 1] do caching in front of the API service with a reverse proxy.
+* [option 2] do caching inside the API service.
+
+## Decision Outcome
+
+Chosen option: "[option 1]", because we have an optional
```
Comment (morucci) [inline doc/adr/0006-cache-es-queries.md:26]: we choose option 2 instead.

File: monocle/webapp.py (hunk 1 above)
Comment (flepied) [reply monocle/webapp.py:153]: I needed to have it called when loaded in all the cases. I don't understand what will be the advantage of having it in a function.

File: doc/adr/0006-cache-es-queries.md (hunk 2 above)
Comment (flepied) [reply doc/adr/0006-cache-es-queries.md:26]: good catch!

File: monocle/webapp.py (hunk 1 above)
Comment (morucci) [reply monocle/webapp.py:153]: ok that's fine yes.

**Pull request merged**